addons/mail/static/tests/mock_server/mail_mock_server.js

```diff
@@ -358,7 +349,7 @@ async function discuss_channel_mute(request) {
     }
     DiscussChannelMember.write([member.id], { mute_until_dt });
     const channel_data = {
-        id: member.channel_id[0],
```

Comment:
oops :)

Reply:
`(search_)read` was replaced `_filter` in `_find_or_create_member_for_self`

(`read` does the `id, display_name` syntax)